### ISSUE TITLE
Implement new FAQs page design

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#834]
The design for the FAQs page is near completion via issue #813; we are still waiting for the illustrations for the question sections and any other adjustments based on content.
![27545377-4db4ea34-5a55-11e7-9658-6f9e78ad9b70](https://user-images.githubusercontent.com/5550969/28551670-c669f548-70a6-11e7-9812-de3858ce46ed.png)

Here is all of the new FAQs content: https://docs.google.com/document/d/1wXy10aqKNaC2Wuz7tqPBX2rog5qNwEH2_qxuVVPeDl8/edit

**Comment by jenmakes** [comment]
@smarziano Here is the FAQ with swapped sections. They do line up — which is a plus, but the column on the left is much longer. Your call. https://cl.ly/1q222G142C0s

**Comment by smarziano** [comment]
Due to the number of questions and length, @jenmakes updated this page's designs:
![screen shot 2017-07-28 at 4 13 29 pm](https://user-images.githubusercontent.com/5550969/28741729-c91a2b3e-73e2-11e7-81a1-36cf6bde1f2b.png)

![screen shot 2017-07-28 at 4 22 47 pm](https://user-images.githubusercontent.com/5550969/28741733-d665b506-73e2-11e7-88de-0bb065f1c631.png)

**Comment by smarziano** [comment]
FAQs are looking great on staging. Here are a few updates that need to be made:
- [ ] First word "Chicago" is missing the C in this question: "What are the characteristics of a high-quality early learning program, such as accreditation?"
- [ ] Update to the steps under this question "What is the application process?" should be:  
1) Visit chicagoearlylearning.org, click on the link to apply to preschool, and create an online account for your family
2) Fill in information on your family, such as employment and income, information on your child, and your top choice preschool sites
3) If your child qualifies, you will be contacted with information on how to verify your child’s preschool seat
- [ ] Remove "Is the online preschool application mobile-friendly?" due to lack of content
- [ ] Remove "Can I change my application after it has been submitted?
If you would like to change aspects of your application after it has been submitted, please call us at 312-744-9322." This is a duplicate of another question and I'm unfamiliar with this number.
- [ ] Add telephone and email hyperlinks under this question "How long do I have to verify my documents after I’ve received an offer?"
- [ ] Add either "Call" text or a phone icon to the Chicago Early Learning hotline button to make it clear that parents will call the hotline

**Comment by smarziano** [comment]
Great work on FAQs. I am going to add that this is waiting on merge.
